src/main/java/springdata/SchedulerService.java

```diff
@@ -0,0 +1,42 @@
+package springdata;
+
+import entity.Person;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class SchedulerService {
+
+    private static final String CRON = "*/10 * * * * *";
+
+    private final PersonService personService;
+    private final EmailService emailService;
+
+    @Scheduled(cron = CRON)
+    public void sendMailToUsers() {
+        LocalDate date = LocalDate.now();
+        int month = date.getMonthValue();
+        int day = date.getDayOfMonth();
+        List<Person> list = personService.findAllByBirthDay(month, day);
+        if (!list.isEmpty()) {
```

Comment:
Do you need this check?

Reply:
yes I need it) Im using example which we have for completion this homework https://habr.com/ru/post/352954/

Reply:
![image](https://user-images.githubusercontent.com/20267148/57581781-1028cf00-74c5-11e9-9c5e-d87c8dab6048.png)
You should not just copy the code. 
Explain why do you need this check?